packages/edit-post/src/components/sidebar/template/actions.js

```diff
@@ -79,20 +84,23 @@ function PostTemplateActions() {
 						onSubmit={ ( event ) => {
 							event.preventDefault();
 							const defaultTitle = __( 'Custom Template' );
-							const templateContent = [
-								createBlock( 'core/site-title' ),
-								createBlock( 'core/site-tagline' ),
-								createBlock( 'core/separator' ),
-								createBlock( 'core/post-title' ),
-								createBlock( 'core/post-content', {
-									layout: { inherit: true },
-								} ),
-							];
+							const newTemplateContent =
+								defaultTemplate ??
+								serialize( [
+									createBlock( 'core/site-title' ),
+									createBlock( 'core/site-tagline' ),
+									createBlock( 'core/separator' ),
+									createBlock( 'core/post-title' ),
+									createBlock( 'core/post-content', {
+										layout: { inherit: true },
+									} ),
+								] );
```

Comment:
Shouldn't the default at least be semantically correct?
Ideally, the `site-title` & `site-tagline` blocks should be inside a `group` block with `tagName=header`, and the `post-title` & `post-content` blocks should be inside a `group` block with `tagName=main` :thinking:

Reply:
I don't mind that change, this is just what we had before

Reply:
Yeah, could be in a separate PR if we don't want to include the change here, it would however be a good improvement :+1: